lucene/core/src/java/org/apache/lucene/util/fst/Builder.java

```diff
@@ -69,6 +69,8 @@
 
   private final IntsRefBuilder lastInput = new IntsRefBuilder();
 
+  final boolean useDirectArcAddressing;
```

Comment:
private final (?)

Reply:
FST reads this so it can't be private. This is following the same pattern as some existing settings like `allowArrayArcs`, which by the way should probably be final but isn't.